lizmap/modules/admin/templates/project_list_zone.tpl

```diff
@@ -15,43 +17,46 @@
 </div>
 {/if}
 
-{if $hasSomeProjectsNotDisplayed}
-<div>
-    {@admin.project.not.displayed@}
```

Comment:
This string is still in the source code. I guess we want to remove it or we should just rephrase ?

Reply:
I used a jMessage to display a warning at the top of the page